release_dashboard/lib/widgets/progression.dart

```diff
@@ -31,7 +32,10 @@ class MainProgression extends StatefulWidget {
     'Apply Engine Cherrypicks',
     'Codesign Engine Binaries',
     'Publish the Release',
-    'Release is Successfully published'
+    'Release is Successfully published',
```

Comment:
Is this not redundant with `'Release is Completed'`?

Reply:
This will get updated in this PR: https://github.com/flutter/cocoon/pull/1483/files
Here: https://github.com/Yugue/cocoon/blob/8e0a209a4660f703ada44797cf9b4976b3178b86/release_dashboard/lib/widgets/progression.dart#L33